docs/providers/aws/events/apigateway.md

```diff
@@ -773,3 +805,116 @@ endpoint of your proxy, and the URI you want to set a proxy to.
 
 Now that you have these two CloudFormation templates defined in your `serverless.yml` file, you can simply run
 `serverless deploy` and that will deploy these custom resources for you along with your service and set up a proxy on your Rest API.
+
+## Share API Gateway and API Resources
+
+As you application grows, you will have idea to break it out into multiple services. However, each serverless project generates new API Gateway by default. If you want to share same API Gateway for muliple projects, you 'll need to reference REST API ID and Root Resource ID into serverless.yml files
```

Comment:
Suggested change:
"As you application grows, you may want to break it out into multiple services. However, each serverless project generates new API Gateway Rest API resource by default. If you want to share a single API Gateway for muliple projects you'll need to reference REST API ID and Root Resource ID into serverless.yml files"